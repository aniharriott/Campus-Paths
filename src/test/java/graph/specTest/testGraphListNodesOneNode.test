# Test Graph listNodes one node file.
# Tests that one node is printed
# when there is one node in the graph.

# Create graph
CreateGraph graph1

# Add node
AddNode graph1 n1

# Print nodes in graph
ListNodes graph1