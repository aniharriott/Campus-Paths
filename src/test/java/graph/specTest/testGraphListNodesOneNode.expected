# Test Graph listNodes one node file.
# Tests that one node is printed
# when there is one node in the graph.

# Create graph
created graph graph1

# Add node
added node n1 to graph1

# Print nodes in graph
graph1 contains: n1